frontend/src/app/signUp/IdentityVerification/utils.ts

```diff
@@ -1,9 +1,6 @@
 import * as yup from "yup";
 
-const EXPERIAN_ANSWER_KEY = "outWalletAnswer";
```

Comment:
Nice to get rid of as many of these confusing Experian implementation details as we can : )